include/sleipnir/control/OCPSolver.hpp

```diff
@@ -0,0 +1,76 @@
+
+
+#pragma once
+
+#include <Eigen/Core>
+
+#include "sleipnir/SymbolExports.hpp"
+#include "sleipnir/optimization/OptimizationProblem.hpp"
+#include "sleipnir/autodiff/VariableMatrix.hpp"
+
+namespace sleipnir {
+
+typedef std::function<VariableMatrix(double, VariableMatrix, VariableMatrix)> FixedStepODE;
+typedef std::function<VariableMatrix(VariableMatrix, VariableMatrix)> FixedStepObjective;
+typedef std::function<void(VariableMatrix, VariableMatrix, bool)> FixedStepConstraintFunction;
```

Comment:
The modern way to write typedefs is like this: `using Name = std::function<things>;`.